client/pages/product/addproduct.jsx

```diff
@@ -15,31 +15,11 @@ const CreateProduct = () => {
     e.preventDefault();
 
     try {
-      const token = window.localStorage.getItem('token');
-
-      const { name, description, image, price, intervals} = product;
-
-      const responseData = await axiosInstance({
-        method: 'POST',
-        url: '/api/products/products',
-        data: {
-          name,
-          description,
-          image,
-          price,
-          intervals
-        },
-        headers: {
-          Authorization: `${token}`,
-        },
-      });
-
+      const responseData = await axiosInstance.post('/api/products/products', product);
```

Comment:
why is the route named `api/products/products`? it's an incorrect name, it has duplicate